src/clients.rs

```diff
@@ -43,6 +47,42 @@ pub async fn get_bundle_upload_location(
         .context("Failed to get repsonse body as json")
 }
 
+pub async fn get_quarantine_bulk_test_status(
+    origin: &str,
+    api_token: &str,
+    org_slug: &str,
+    repo: &Repo,
+    test_identifiers: &[Test],
+) -> anyhow::Result<QuarantineBulkTestStatus> {
+    let client = reqwest::Client::new();
+    let resp = match client
+        .post(format!("{}/v1/metrics/getQuarantineBulkTestStatus", origin,))
```

Comment:
```suggestion
        .post(format!("{}/v1/metrics/getQuarantineBulkTestStatus", origin))
```

Reply:
Good catch, we should add trunk check to this repo so it's caught there.